core-plugins/docs/HDFSAction-action.md

```diff
@@ -0,0 +1,53 @@
+# HDFS Action
+
+
+Description
+-----------
+Moves file(s) within HDFS.
+
+
+Use Case
+--------
+This action can be used when files needs to be moved to a new location in HDFS for the purpose of archiving,
+for example.
+
+
+Properties
+----------
+**sourcePath:** The full HDFS path of the file/directory that the user wants to move. In the case of a directory,
+if fileRegex is set, then all files in the directory matching the wildcard regex will be moved. Otherwise,
+all files in the directory will be moved. Ex: hdfs://hostname/tmp
```

Comment:
```
**sourcePath:** The full HDFS path of the file or directory that is to be moved. In the case of a directory,
 if fileRegex is set, then only files in the source directory matching the wildcard regex will be moved. 
Otherwise, all files in the directory will be moved. For example: `hdfs://hostname/tmp`.
```